components/network-identifier.tsx

```diff
@@ -1,39 +1,31 @@
 import React, {useContext, useEffect, useState} from 'react';
 import {ApplicationContext} from '@contexts/application';
-import {BeproService} from '@services/bepro-service';
 import Indicator from '@components/indicator';
 import {changeNetwork} from '@reducers/change-network';
-import WrongNetworkModal from '@components/wrong-network-modal';
+import {identifierNeworkLabel} from '@helpers/metamask'
 
 const networkMap = {
-  ethereum: `#29b6af`,
-  roptsen: `#ff4a8d`,
-  kovan: `#9064ff`,
-  rinkeby: `#f6c343`,
-  goerly: `#f6c343`
+  Mainnet: `#29b6af`,
+  Ropsten: `#ff4a8d`,
+  Kovan: `#9064ff`,
+  Rinkeby: `#f6c343`,
+  Goerli: `#f6c343`
 }
 
 export default function NetworkIdentifier() {
-  const {state: {currentAddress, network}, dispatch} = useContext(ApplicationContext);
+  const {state: {network}, dispatch} = useContext(ApplicationContext);
 
   function updateNetwork() {
-    if (!currentAddress)
-      return;
-
-    BeproService.bepro.web3.eth.net.getNetworkType()
-                .then(net => {
-                  dispatch(changeNetwork(net));
-                });
+      dispatch(changeNetwork(window.ethereum.networkVersion))
   }
 
-  useEffect(updateNetwork, [currentAddress]);
+  useEffect(updateNetwork, [window.ethereum.networkVersion]);
 
   return network &&
       <>
-        <div className="d-inline-flex align-items-center justify-content-center bg-white py-1 px-2 mr-1 rounded text-uppercase smallCaption fs-smallest text-center text-black text-nowrap"> 
-          <Indicator bg={networkMap[network]} /> <span>{network} {network !== `ethereum` && `testnet` || `mainnet`}</span>
+        <div className="d-inline-flex align-items-center justify-content-center bg-white py-1 px-2 mr-1 rounded text-uppercase family-bold fs-smallest text-center text-black text-nowrap"> 
+          <Indicator bg={networkMap[identifierNeworkLabel(network)]} /> <span>{identifierNeworkLabel(network)} {(network !== '1' && network !== '1284') && `testnet`}</span>
```

Comment:
set the result of `identifierNeworkLabel()` on a variable so the code is easily read, `!== 1` doesn't say much, `!== 'ethereum'` explains exactly what we want to achieve